index.js

```diff
@@ -124,7 +142,51 @@ function maybeSignWebBundle(webBundle, opts, infoLogger) {
   return signedWebBundle;
 }
 
-function validateIWAOptions(opts) {
+function checkIwaOverrideHeaders(headers) {
+  for (const iwaHeaderName of Object.keys(invariableIwaHeaders)) {
+    if (headers[iwaHeaderName] !== invariableIwaHeaders[iwaHeaderName]) {
+      throw new Error(
+        `For Isolated Web Apps ${iwaHeaderName} should be ${invariableIwaHeaders[iwaHeaderName]}. Now it is ${headers[iwaHeaderName]}. If you are bundling a non-IWA, set integrityBlockSign { isIwa: false } in your plugins configs.`
+      );
+    }
+  }
+
+  // TODO: Parse and check `Content-Security-Policy` value.
+  const cspHeaderName = Object.keys(csp)[0];
+  if (!headers[cspHeaderName]) {
+    throw new Error(
+      `For Isolated Web Apps ${cspHeaderName} should not be empty. Default value can be used: ${JSON.stringify(
```

Comment:
Can you make this a bit more explicit? `"For Isolated Web Apps, ${CSP_HEADER_NAME} must have the following minimal strictness: ${JSON.stringify(...)} ..."`